package.json

```diff
@@ -6,14 +6,15 @@
     "init": "opam switch create . 5.1.1 -y --deps-only && npm run install-opam-npm",
     "install-opam-npm": "npm install && npm run install-opam && npm run check-npm-deps",
     "dune": "opam exec -- dune",
-    "build": "npm run dune -- build",
+    "build": "npm run dune -- build @melange",
```

Comment:
Is this necessary? `melange` alias should be added by default to `all`.

Reply:
It's no longer needed if I put `@melange` as dependency for test rule stanzas